lib/intercom_flutter.dart

```diff
@@ -27,22 +27,22 @@ class Intercom {
   }
 
   static Future<dynamic> registerIdentifiedUser({String userId, String email}) {
-    if (userId.isNotEmpty) {
```

Comment:
Maybe we should also check if the `String` is empty.

Reply:
Solved using `userId?.isNotEmpty ?? false`, which checks if `userId` is not empty only if it's not null, otherwise the logical expression is evaluated to false.